src/components/divider/divider.tsx

```diff
@@ -0,0 +1,51 @@
+import { useTheme } from 'native-base';
+import React from 'react';
+import { View, ViewStyle, StyleProp } from 'react-native';
+
+import { useDividerStyles } from './divider.styles';
+
+export interface DividerProps {
+  orientation?: 'horizontal' | 'vertical';
+  thickness?: number;
+  color?: string;
+  style?: StyleProp<ViewStyle>;
```

Comment:
It should not accept color, style - defeats creating an abstraction as a component.

Do you see other popular libs expose this?